### DISCUSSION TITLE
Light mode & dark mode images not displayed in tables on mobile

### DISCUSSION THREAD

**Discussion by dieugab** [community/community#8266] [Mobile] [answered]
Like the title states, images using the light and dark mode suffix don't render in the mobile. Does work in the browser. Example here: https://github.com/gvssr/home-assistant-config#hardware

iOS (app version 1.42.0):
![4acf12e5-6505-40e3-bbac-ecf67c37ed87](https://user-images.githubusercontent.com/61377476/144083316-73b10131-755d-45cf-86e7-39cf5ad636a2.jpg)

Browser:
![1e03179c-f048-4fc5-bf37-165babc582c2](https://user-images.githubusercontent.com/61377476/144083321-26f8c4f5-3a01-4c11-bef6-fbd051be537d.jpg)

**Comment by brianlovin** [comment]
Thanks for the heads up on this @gvssr, and apologies for the slow reply. The team is taking a look at this to see if we can find a fix.

**Comment by chesterbr** [comment, marked as answer]
@gvssr Thank you for your patience. Please check again - it should be working now!

**Reply by dieugab** [reply]
It's working now, my thanks to the team!
